state/index.js

```diff
@@ -13,16 +13,29 @@ const toDict = (array, keyFn) => array.reduce((obj, item) => ({
   [keyFn(item)]: item,
 }), {});
 
+const getCategories = badgers => (
+  Object.keys(badgers
+    .reduce((uniqueCategories, badger) => (
+      badger.categories
+        .reduce((categories, category) => (
+          categories[category] ? categories : { ...categories, [category]: 1 }
+        ), uniqueCategories)
+    ), {}))
+);
```

Comment:
Moved into `badger-brain` fetcher (as you suggested below) so that it matches `selectValidEvents` with events